metallb/overlays/ocp-staging/addresspool.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: metallb.io/v1alpha1
+kind: AddressPool
+metadata:
+  name: csail-10
+spec:
+  protocol: layer2
+  addresses:
+  - 128.52.60.50-128.52.60.60
```

Comment:
I am reserving 10 IPs for the staging cluster. Should be more than enough I think?

https://github.com/CCI-MOC/documentation/pull/108/files